source/_integrations/fitbit.markdown

```diff
@@ -13,84 +13,41 @@ ha_integration_type: integration
 
 The Fitbit sensor allows you to expose data from [Fitbit](https://fitbit.com/) to Home Assistant.
 
-Prior to enabling the sensor, your Home Assistant instance must be enabled with SSL, using services such as [Home Assistant cloud](/cloud/) or [Duck DNS](/integrations/duckdns/) integrating Let's Encrypt. Fitbit will not allow insecure integrations.
+## Prerequisites
 
-<div class='note'>
-There is no need for Access from the Internet to Home Assistant for this Integration to work. Home Assistant can only be accessible from the internal Network or via VPN. But a Certificate is required in any case.
-</div>
+You need to configure developer credentials to allow Home Assistant to access your Fitbit account.
 
-Your Fitbit account must also be registered as an Developer account at the [Fitbit Developer Portal](https://dev.fitbit.com), and have a verified email address. 
+{% details "Generate Client ID and Client Secret" %}
 
-Enable the sensor by adding the following to your `configuration.yaml` file:
+1.  Your Fitbit account must be registered as an Developer account at the [Fitbit Developer Portal](https://dev.fitbit.com), and have a verified email address. 
+2. Visit [Register an Application](https://dev.fitbit.com/apps/new)
+3. Enter an *Application Name* of your chosing, for example **Home Assistant**.
+4. Since we are creating a *Personal* registration, the details for *Description*, *Application Website URL*, *Organization* etc must be filled in, however the contents do not matter and will only be shown to you on the authorization page later.
```

Comment:
```suggestion
4. Since we are creating a *Personal* registration, the details for **Description**, **Application Website URL**, **Organization**, etc. must be filled in. However, the contents do not matter and will only be shown to you on the authorization page later.
```